packages/core-mobile/app/new/features/buyOnramp/hooks/useBuy.ts

```diff
@@ -0,0 +1,50 @@
+import { useRouter } from 'expo-router'
+import { useCallback } from 'react'
+import { useSelector } from 'react-redux'
+import { selectIsMeldIntegrationBlocked } from 'store/posthog'
+import { useOnRampToken } from '../store'
+import { CryptoCurrency } from './useSearchCryptoCurrencies'
+
+type NavigateToBuyParams = {
+  showAvaxWarning?: boolean
```

Comment:
I don't think we need showAvaxWarning anymore since we are supporting buying more than just avax now.